lib/elixir/lib/kernel.ex

```diff
@@ -1206,13 +1228,22 @@ defmodule Kernel do
   @doc """
   Gets the element at the zero-based `index` in `tuple`.
 
+  It raises `ArgumentError` when `tuple` is empty,
+  or when `index` is a negative number or it is out of range of the tuple elements.
```

Comment:
@ericmj could you please quote how the phrase should be, cause I think I misunderstood your initial comment